core/admin/mailu/api/v1/user.py

```diff
@@ -242,8 +242,7 @@ def patch(self, email):
         if 'forward_enabled' in data:
             user_found.forward_enabled = data['forward_enabled']
         if 'forward_destination' in data and len(data['forward_destination']) > 0:
```

Comment:
Why do we not change forward destination when the length is zero?

Shouldn't the zero test also be removed?

Is it possible to delete the forward destination with this test in place?

Reply:
What purpose would have setting the `forward_destination` to an empty list? Disable forwarding could be achieved by disabling it. So the only reason I can think of right now is to remove a previously configured destination to "hide" where it was forwarded to when it was active.

Ofc, the data is not needed when forwarding is disabled, so technically, we could also remove the zero-length check, but I'm not involved enough to know why it was added.